index.html

```diff
@@ -4,13 +4,38 @@
 <head>
```

Comment:
I think if you switch to this new api and then work on some styling, your work would be done !